sw/host/opentitanlib/src/spiflash/flash.rs

```diff
@@ -311,4 +319,10 @@ impl SpiFlash {
         }
         Ok(())
     }
+
+    /// Send the software reset sequence to the `spi` target.
+    pub fn chip_reset(spi: &dyn Target) -> Result<()> {
+        spi.run_transaction(&mut [Transfer::Write(&[Self::RESET])])?;
```

Comment:
My original error: this should be two transactions:
```
spi.run_transaction(&mut [Transfer::Write(&[Self::RESET_ENABLE])])?;
spi.run_transaction(&mut [Transfer::Write(&[Self::RESET])])?;
```

Reply:
Not really, I removed the RESET_ENABLE since we used a different opcode, reverted this change.